modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -146,14 +137,7 @@ export default {
         message: 'Uploaded media will appear here',
         emoji: '🖼'
       },
-      confirmingDiscard: false,
-      discardConfirmed: false,
```

Comment:
This is still part of a conditional on L255

Reply:
Removed.